app/services/analytics_events.rb

```diff
@@ -2376,69 +2376,121 @@ def user_registration_email_confirmation(
     )
   end
 
-  # Tracks exceptions that are raised when running GetUspsProofingResultsJob
-  # @param [String] reason why was the exception raised?
-  # @param [String] enrollment_id
+  # Tracks if USPS in-person proofing enrollment request fails
+  # @param [String] context
+  # @param [String] reason
+  # @param [Integer] enrollment_id
   # @param [String] exception_class
   # @param [String] exception_message
-  def idv_in_person_usps_proofing_results_job_exception(
-    reason:, enrollment_id:, exception_class: nil, exception_message: nil, **extra
+  def idv_in_person_usps_request_enroll_exception(
+    context:,
+    reason:,
+    enrollment_id:,
+    exception_class:,
+    exception_message:,
+    **extra
   )
     track_event(
-      'GetUspsProofingResultsJob: Exception raised',
-      reason: reason,
+      'USPS IPPaaS enrollment failed',
+      context: context,
       enrollment_id: enrollment_id,
       exception_class: exception_class,
       exception_message: exception_message,
+      reason: reason,
       **extra,
     )
   end
 
-  # Tracks individual enrollments that fail during GetUspsProofingResultsJob
-  # @param [String] reason why did this enrollment fail?
-  # @param [String] enrollment_id
-  def idv_in_person_usps_proofing_results_job_enrollment_failure(reason:, enrollment_id:, **extra)
+  # GetUspsProofingResultsJob is beginning. Includes some metadata about what the job will do
+  # @param [Integer] enrollments_count number of enrollments eligible for status check
+  # @param [Integer] reprocess_delay_minutes minimum delay since last status check
+  def idv_in_person_usps_proofing_results_job_started(
+    enrollments_count:,
+    reprocess_delay_minutes:,
+    **extra
+  )
     track_event(
-      'GetUspsProofingResultsJob: Enrollment failed proofing',
-      reason: reason,
-      enrollment_id: enrollment_id,
+      'GetUspsProofingResultsJob: Job started',
+      enrollments_count: enrollments_count,
+      reprocess_delay_minutes: reprocess_delay_minutes,
+      **extra,
+    )
+  end
+
+  # GetUspsProofingResultsJob has completed. Includes counts of various outcomes encountered
+  # @param [Float] duration number of minutes the job was running
```

Comment:
I think the description is helpful to clarify the unit of time, but I expect it could be hard to interpret if somewhere were looking at a log query result ("How long is `duration: 0.5`?"). Maybe we should include the unit of time in the property name itself, e.g. `duration_minutes` or `duration_in_minutes`.

Also, how long do these jobs usually run? I might think a smaller unit of time could be easier to avoid dealing with precision, like seconds.

Reply:
Yeah fair points. Updated in eb2f0afe8